include/RAJA/config.hpp.in

```diff
@@ -277,6 +277,18 @@ const int DATA_ALIGN = @RAJA_DATA_ALIGN@;
 #cmakedefine RAJA_HAVE_ALIGNED_ALLOC
 #cmakedefine RAJA_HAVE_MM_MALLOC
 
+template<typename T>
+RAJA_INLINE
+T * align_hint(T * x)
```

Comment:
Maybe I'm confused, but I don't know where this is being used?

Reply:
Currently just in a unit test. But the issue is that when adding alignment hints, GCC and clang will return a void * while Intel will return nothing. This allows a common interface for alignment memory hints.

Reply:
In the scenario that CUDA is enabled, I opted for removing compiler alignment hints. I am not sure what the best practice would be here.